### PR TITLE
pip.sh: die on pip install failure

### PR DESCRIPTION
**Pull request by caisq**

### CODE REVIEW

File: tensorflow/tools/ci_build/builds/pip.sh [hunk 1]
```diff
@@ -141,8 +142,10 @@ echo "Installing pip whl file: ${WHL_PATH}"
 
 # Call pip install twice, first time with --upgrade and second time without it
 # This addresses the sporadic test failures related to protobuf version
-${PYTHON_BIN_PATH} -m pip install -v --user --upgrade ${WHL_PATH} numpy==1.8.2 &&
-${PYTHON_BIN_PATH} -m pip install -v --user ${WHL_PATH} &&
+${PYTHON_BIN_PATH} -m pip install -v --user --upgrade ${WHL_PATH} numpy==1.8.2 \
+|| die "pip install (1st step) FAILED"
```
Comment (jendap) [inline tensorflow/tools/ci_build/builds/pip.sh]: nit: you can put here "...(1st step - pip install --upgrade)..." or something like that (one may not remember which is is 1st and which is 2nd)

**Comment by jendap** [comment]
@tensorflow-jenkins test this please

**Comment by caisq** [comment]
@jendap's comment addressed. Squashed.

**Comment by vrv** [comment]
Merged

**Pull request closed without merging**